pkg/controllers/jobflow/jobflow_controller_util.go

```diff
@@ -29,8 +29,8 @@ func getJobName(jobFlowName string, jobTemplateName string) string {
 	return jobFlowName + "-" + jobTemplateName
 }
 
-// GetTemplateString get the JobTemplate information string
-func GetTemplateString(namespace, name string) string {
+// GenerateObjectString generates the object information string using namespace and name
+func GenerateObjectString(namespace, name string) string {
```

Comment:
As I recall, in k8s, this function has a specific name `NamespacedName`, so you can name this func GetNamespacedName()

Reply:
https://github.com/kubernetes/kubernetes/blob/9d63e575f8482b857ef328325d2107df840b9deb/pkg/scheduler/framework/types.go#L959

But this func is `namespace/name`, different from your code `namespace.name`. Maybe you should change the code?

Reply:
I know there is such a method, but my idea is that this label has been around for a while, and I am not sure if there are any users who would use this label. So I think it is better to keep it as it is.

Reply:
I just changed the method name, the logic is the same as the original code

Reply:
ok that's no problem